proxy/attempt-register-vm-on-proxy.sh

```diff
@@ -73,8 +79,13 @@ echo "Backend id: ${BACKEND_ID}"
 
 # Store the registration information in a ConfigMap
 kubectl create configmap inverse-proxy-config \
+        --namespace=${NAMESPACE}
```

Comment:
I don't think this is necessary because kubectl will create the configmap in the same namespace. 
The proxy is running under a role not a clusterrole that only allows it to create configmap in the same ns. 
if it creates in a different ns the rbac will reject it. 
https://github.com/kubeflow/pipelines/blob/master/manifests/kustomize/base/proxy/proxy-role.yaml